src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -964,6 +964,9 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('content_type', arg_group='Additional Flags', help="Specify content type of the file. ")
         c.argument('follow_symlinks', arg_group='Additional Flags', action='store_true',
                    help='Follow symbolic links when uploading from local file system.')
+        c.argument('cap_mbps', arg_group='Additional Flags', help="Caps the transfer rate, in megabits per second. "
+                   "Moment-by-moment throughput might vary slightly from the cap. "
+                   "If this option is set to zero, or it is omitted, the throughput isn't capped. ")
```

Comment:
Do we need a validator to check if it's valid number? At least it needs to be a number.

Reply:
`azcopy` already has a validator to check, do we need to check it again?